test/nlp_program.jl

```diff
@@ -627,6 +632,125 @@ function test_compute_derivatives_Finite_Diff(;
     end
 end
 
+################################################
+#=
+# Test Objective Sensitivity wrt Parameters
+=#
+################################################
+
+function test_ObjectiveSensitivity()
+    # Model 1
+    model = Model(() -> DiffOpt.diff_optimizer(Ipopt.Optimizer))
+    set_silent(model)
+
+    # Parameters
+    @variable(model, p ∈ MOI.Parameter(1.5))
+    @variable(model, p_prox)
+
+    # Variables
+    @variable(model, x)
+
+    # Constraints
+    @constraint(model, con, p_prox == p) # dual fishing :)
+    @constraint(model, x * sin(p_prox) == 1)
```

Comment:
this would not work if the parameter was directly inside the sine?

Reply:
yes, ran this to get the results from the duals but now set it as a value test